.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: node_js
+env:
+- COVERAGE=
```

Comment:
How so? It looks to me like it will do one run of the tests without coverage and one with coverage.

However, I wonder if it matters. Could we just do one run of the tests with coverage enabled?